src/js/containers/Footer.jsx

```diff
@@ -130,7 +130,7 @@ const Footer = ({
                                 </li>
                                 <li>
                                     <FooterExternalLink
-                                        link="https://github.com/fedspendingtransparency/usaspending-website/releases"
+                                        link="https://github.com/fedspendingtransparency/usaspending-website/wiki#may-4-2021"
```

Comment:
```suggestion
                                        link="https://github.com/fedspendingtransparency/usaspending-website/wiki"
```
Sorry I should have caught this in the AC! Your change matches the ticket, but I don't think we want to hard code a date in the url (the latest release will be at the top of the page)